lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -23,13 +32,13 @@ def self.instances
   end
 
   def exists?
-    PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task.new.exists? resource[:name]
+    task_scheduler_helper.new.exists? resource[:name]
   end
 
   def task
     return @task if @task
 
-    @task ||= PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task.new
+    @task ||= task_scheduler_helper.new
     @task.activate(resource[:name] + '.job') if exists?
```

Comment:
Possible ongoing issues with the appending of '.job'

Reply:
I think in this case it was just because tests weren't updated from `TaskScheduler2V1Task` to `TaskScheduler2Task`... see https://github.com/puppetlabs/puppetlabs-scheduled_task/pull/27/commits/b73f8c3282401f9575ba4208875c283f1c34583c